I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script00.rpy", line 560, in script call
    call dungeon
  File "game/script00.rpy", line 641, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 689, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 693, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 560, in script call
    call dungeon
  File "game/script00.rpy", line 641, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 689, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 693, in script call
    call screen dungeon_run()
ScriptError: could not find label 'endRun'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/bootstrap.py", line 289, in bootstrap
    renpy.main.main()
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/main.py", line 357, in main
    run(restart)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/main.py", line 77, in run
    renpy.execution.run_context(True)
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/execution.py", line 598, in run_context
    context.run()
  File "game/script00.rpy", line 560, in script call
    call dungeon
  File "game/script00.rpy", line 641, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 689, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 693, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 560, in script call
    call dungeon
  File "game/script00.rpy", line 641, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 689, in script call
    call screen dungeon_run()
  File "game/script00.rpy", line 693, in script call
    call screen dungeon_run()
  File "/home/digiholic/workspace/SummonerSweetheart-0.9-all/renpy/script.py", line 595, in lookup
    raise ScriptError("could not find label '%s'." % str(label))
ScriptError: could not find label 'endRun'.

Linux-3.11.0-26-generic-x86_64-with-debian-wheezy-sid
Ren'Py 6.18.3.761
Summoner Sweetheart 0.9
